stravalib/client.py

```diff
@@ -510,15 +566,20 @@ def get_activity(self, activity_id, include_all_efforts=False):
         )
         return model.Activity.parse_obj({**raw, **{"bound_client": self}})
 
-    def get_friend_activities(self, limit=None):
-        """
-        DEPRECATED This endpoint was removed by Strava in Jan 2018.
+    # TODO: can we remove commented out code in this function?
```

Comment:
@jsamoocha there is commented code in this method and in another one in this module. do you think we can just delete it or is it a reminder for something else that could be converted to a TODO ?

Reply:
This whole method should be removed as it is no longer part of the Strava API. We should systematically check this sometime; see #323.

So, yes, the commented-out code can be deleted.

Reply:
ok i'll remove commented code here and leave a todo that this entire method can be removed. i think given this pr will get big with the black fix, and this is more of a cleanup pr, let's implement the removal of methods separately in another pr.